Jenkinsfile_CNP

```diff
@@ -9,6 +9,13 @@ def component = "user-profile-api"
 def secrets = [
         's2s-${env}': [
                 secret('microservicekey-rd-user-profile-api', 'S2S_SECRET')
+        ],
+        'rd-${env}': [
+                secret('user-profile-api-POSTGRES-HOST', 'FUNC_DATABASE_HOST'),
+                secret('user-profile-api-POSTGRES-PASS', 'FUNC_DATABASE_PASS'),
+                secret('user-profile-api-POSTGRES-USER', 'FUNC_DATABASE_USER'),
+                secret('user-profile-api-POSTGRES-DATABASE', 'FUNC_DATABASE_NAME'),
```

Comment:
could you please add the entries from jenkins_nightly file as well ?

Reply:
also add the entries only for aat environment like PRD script to maintain uniformity?